gitlab/v4/objects/topics.py

```diff
@@ -29,3 +31,39 @@ class TopicManager(CRUDMixin, RESTManager):
 
     def get(self, id: Union[str, int], lazy: bool = False, **kwargs: Any) -> Topic:
         return cast(Topic, super().get(id=id, lazy=lazy, **kwargs))
+
+    @cli.register_custom_action(
+        "TopicManager",
+        mandatory=("source_topic_id", "target_topic_id"),
+    )
+    @exc.on_http_error(exc.GitlabMRClosedError)
+    def merge(
+        self,
+        source_topic_id: Union[int, str],
```

Comment:
The docs say it only allows an integer.

Reply:
I think it just needs to be IDs, as long as the strings contain the IDs this will work: `gl.topics.merge("1", "2")`. I can change it, in the end it's just hints but technically it will be a bit too strict :)

Reply:
Makes sense. Thanks.